app-android/tutashared/src/main/java/de/tutao/tutashared/BuildConfig.kt

```diff
@@ -0,0 +1,30 @@
+package de.tutao.tutashared
+
+import android.util.Log
+
+data class BuildConfig(
```

Comment:
is there no way we can generate the same thing for the shared module?

Reply:
Done, the only thing is that the versionName must be filled manually, what shouldn't be a problem since we use the bumpVersion script